tests/automation-framework/rancherclient/config.go

```diff
@@ -0,0 +1,94 @@
+package rancherclient
+
+type Config struct {
+	CattleTestURL     string `json:"cattleTestURL"`
+	AdminToken        string `json:"adminToken"`
+	UserToken         string `json:"userToken,omitempty"`
+	CNI               string `json:"cni" default:"calico"`
+	KubernetesVersion string `json:"kubernetesVersion"`
+	NodeRoles         string `json:"nodeRoles,omitempty"`
+	DefaultNamespace  string `default:"fleet-default"`
+	Insecure          *bool  `json:"insecure" default:"true"`
+	CAFile            string `json:"caFile" default:""`
+	CACerts           string `json:"caCerts" default:""`
+}
+
+func (r *Config) SetCattleTestURL(url string) {
```

Comment:
why do we have all these getters and setters?

Reply:
I don't have a particular reason. I'll remove them.